sampleFiles/clinical/chemotherapy.tsv

```diff
@@ -1,4 +1,4 @@
 program_id	submitter_donor_id	submitter_treatment_id	chemotherapy_drug_name	chemotherapy_dosage_units	cumulative_drug_dosage
-TEST-CA	ICGC_0002	T_02	Still need it	mg/m²	4
-TEST-CA	ICGC_0003	T_03	Need list	µg/m²	20
-TEST-CA	ICGC_0001.1	T_04	Need list	µg/m²	35
\ No newline at end of file
+TEST-CA	ICGC_0002	T_02	Allopurinol	mg/m²	4
```

Comment:
does this work ?

Reply:
Hmm, guess it doesn't work with the sample schema or the ones in dev/qa eh. I was just working with this to create a file for int-test, must have committed it. Will revert since it doesn't match those schema